torch/distributed/pipelining/stage.py

```diff
@@ -935,6 +935,67 @@ def _validate_fwd_outputs(self, outputs: tuple[torch.Tensor, ...]):
             f"Stage {self.stage_index} forward outputs", expected_tensors_meta, outputs
         )
 
+    def _init_p2p_neighbors(self, ops: list[dist.P2POp]) -> list[dist.P2POp]:
+        """
```

Comment:
Nit: name implies action, but function just returns ops. Call it get_.._ops() instead? Or put the batchp2p and wait inside?